lib/active_model/serializer/associations.rb

```diff
@@ -28,8 +27,8 @@ class << base
       end
 
       module ClassMethods
-        def inherited(base)
-          base._reflections = self._reflections.try(:dup) || []
+        def inherit_associations(base)
```

Comment:
I'm always ok with small bug fixes here and there. Esp if they are stumbled upon, and aren't documented anywhere. Better to catch it early before it becomes an actual problem.